projects/reservoir/fv3/era_process.py

```diff
@@ -0,0 +1,177 @@
+import xarray as xr
+import glob
+import numpy as np
+import vcm
+import vcm.catalog
+from vcm.catalog import catalog
+import xarray as xr
+import netCDF4 as nc
+import glob
+import cftime
+import datetime
+import os
+
+def main():
+    path = '/home/paulah/data/era5/'
+    variables = ['v_wind']
+    setup_fregrid()
+    for var in variables:
+        check_time_steps_complete(path, var)
+        weekdays = ['mon','tue','wed','thu','fri','sat','sun']
+        for i, day in enumerate(weekdays):
+            merge_into_file(path,var,i,day)
+            print(day,' is merged.')
+            regridded = regrid_to_cubed_sphere(path,var,day)
+            print(day,' is regridded.')
+            coarsened = coarsen(var,day)
+            print(day,' is coarsened.')
+            masked = mask(coarsened)
+            print(day,' is masked.')
+            masked.to_netcdf(path + var + '/masked/'+var+'_'+day+'.nc') 
+            print(day,' is done.')
+            
+def setup_fregrid():
+    #os.system("rm -rf fregrid-example")
+    os.system("mkdir -p fregrid-example/source-data")
+    os.system("mkdir -p fregrid-example/source-grid-data")
+    os.system("gsutil -m cp gs://vcm-ml-intermediate/2023-06-19-fregrid-example-data/*.nc fregrid-example/source-data/")
+    os.system("gsutil -m cp gs://vcm-ml-raw/2020-11-12-gridspec-orography-and-mosaic-data/C96/*.nc fregrid-example/source-grid-data/")
```

Comment:
I'm not too familiar with the regridding/coarsening debugging that you had to do, but is there a reason why this workflow first puts the data on the C96 grid and then coarsens to C48, as opposed to just going to C48 directly and skipping the coarsening step?

Reply:
I think you could skip the coarsening step and replace this path with the one containing equivalent files for C48 gridspec data gs://vcm-ml-raw/2020-11-12-gridspec-orography-and-mosaic-data/C48/*.nc

Reply:
Thanks for that, I changed to the c48 grid